### PR TITLE
revamp docs to include getting started section

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Install
+=======
+
+Install the library by adding it manually to your ``composer.json``:
+
+.. code-block:: json
+
+   {
+     "require": {
+       "crate/crate-pdo":"~0.6.0"
```
Comment (chaudum) [inline docs/getting-started.rst]: Current version is 0.7.0 (https://github.com/crate/crate-pdo/commit/074f7af649f62f8070818491636f7e0087b90be8).

File: docs/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+.. code-block:: php
+
+   require_once __DIR__.'/vendor/autoload.php';
+   use Crate\PDO\PDO as PDO;
+
+   $dsn = 'crate:SERVER_IP:4200';
+   $connection = new PDO($dsn, null, null, null);
+
+As CrateDB doesn't support authentication, the other parameters can be left
```
Comment (chaudum) [inline docs/getting-started.rst]: This is not correct any more.

File: docs/getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+source name string (`dsn <https://en.wikipedia.org/wiki/Data_source_name>`_)
+and then connect to it.
+
+.. code-block:: php
+
+   require_once __DIR__.'/vendor/autoload.php';
+   use Crate\PDO\PDO as PDO;
+
+   $dsn = 'crate:SERVER_IP:4200';
+   $connection = new PDO($dsn, null, null, null);
```
Comment (chaudum) [inline docs/getting-started.rst]:
```php
$connection = new PDO($dsn, $user, $password, $options);
```

**Comment by nslater** [comment]
@chaudum can you re-check please?

**Pull request merged**